### DISCUSSION TITLE
How to Implement BuyNow on a Individual Product

### DISCUSSION THREAD

**Discussion by ramkishore-yerubandi** [vendurehq/vendure#1419] [Q&A]
I am looking for suggestions to implement BuyNow on an Individual Product bypassing add to cart of the item.

**Comment by michaelbromley** [comment]
Hi,
Do still want the purchase to be recorded in Vendure? If you bypass "add to cart" then you won't get an Order. Maybe you can describe the desired workflow and results a bit more?

**Reply by ramkishore-yerubandi** [reply]
On Product, I should support both 'Add to Cart' and 'Buy Now'. When a user clicks on Buy Now on a product, we should have a ability to directly lead the user to the Payment Option for that Product only even if he has other items in the cart. Like the way Amazon works

**Reply by michaelbromley** [reply]
OK. This would take some significant custom work to support, since Vendure by default only supports a single active Order at a time. You'd need to do something like "stash" the active order in a new state, create a new order for the single product, and once that is checked out, "unstash" the original order.
